python/python.bzl

```diff
@@ -12,26 +12,44 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+"""Re-exports for some of the core Bazel Python rules.
```

Comment:
Can/should we follow a similar pattern as [skylib's old `lib.bzl`](https://github.com/thomasvl/bazel-skylib/blob/79c9da076f7b869b1c09b2f7dd16b2458a34f2f2/lib.bzl#L17-L21) and `print()` a warning when this file is used?

I know it can get a bit noisy, but otherwise I suspect no one will see these deprecation warnings.